core/src/main/java/de/gerrygames/viarewind/protocol/protocol1_7_6_10to1_8/packets/WorldPackets.java

```diff
@@ -248,8 +248,7 @@ public void handle(PacketWrapper packetWrapper) throws Exception {
 					@Override
 					public void handle(PacketWrapper packetWrapper) throws Exception {
 						for (int i = 0; i < 4; i++) {
-							String line = packetWrapper.read(Type.STRING);
-							line = ChatUtil.jsonToLegacy(line);
+							String line = ChatUtil.jsonToLegacy(packetWrapper.read(Type.COMPONENT));
```

Comment:
this needs a change in VV as well (https://github.com/ViaVersion/ViaVersion/commit/be6e086aacd3769a600ba9b5850e212fe29eea41#diff-91a2f12fc625f86096e049c203e360e96c851d61bd6c73d787894ce7139a93a9R48)

Reply:
Ohh I only searched in ViaBackwards. But I guess we should change it for consistency.

Reply:
Hm I will change this back to the string type.